tests/unit/plugin.spec.ts

```diff
@@ -0,0 +1,35 @@
+/**
+ * @copyright Copyright (c) 2023 Ferdinand Thiessen <opensource@fthiessen.de>
+ *
+ * @author Ferdinand Thiessen <opensource@fthiessen.de>
+ *
+ * @license AGPL-3.0-or-later
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+import { NextcloudVuePlugin } from '../../src/plugin'
+import { createLocalVue, mount } from '@vue/test-utils'
+
+describe('Nextcloud Vue Plugin', () => {
+	'use strict'
+
+	it('can be installed', () => {
+		const localVue = createLocalVue()
+		localVue.use(NextcloudVuePlugin)
+		const wrapper = mount({ render: (h) => h('NcButton') }, { localVue })
+		expect(wrapper.vm.$el.tagName).toBe('BUTTON') // If it could no be installed it would be 'NCBUTTON'
```

Comment:
This test also tests that `NcButton` component renders `<button>` which in theory is not guaranteed and not the result of installing the plugin. 

> // If it could not be installed, it would be 'NCBUTTON'

If that is true, can we test `.not.toBe('NCBUTTON')` as it's said?

Or can we test it with `findComponent` or `wrapper.vm`?

Reply:
> his test also tests that NcButton component renders <button> which in theory is not guaranteed 

True, assuming button is the default, testing the negative version (not ncbutton) should be safer as it does not assume anything.

> If that is true, can we test .not.toBe('NCBUTTON') as it's said?

Yes :)